File: C/URI 1008.c
#include <stdio.h>
int main(){
  int NUMBER,horas;
  scanf("%d %d",&NUMBER,&horas);
  float valor;
  scanf("%f",&valor);
  float SALARY;
  SALARY = horas * valor;
  printf("NUMBER = %d\n",NUMBER);
  printf("SALARY = U$ %.2f\n",SALARY);
  return 0;
}

File: Java/URI 1008.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      int num, horas;
      double valor,salario;
      num = sc.nextInt();
      horas = sc.nextInt();
      valor = sc.nextDouble();
      salario = valor * horas;
      System.out.println("NUMBER = " + num);
      System.out.printf("SALARY = U$ %.2f%n", salario);
      sc.close();
  }
}